battleship-game/src/router.js

```diff
@@ -0,0 +1,45 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+
+import Home from './components/Home.vue';
+import Board from './components/Board.vue';
+import Configurations from './components/Configurations.vue';
+import BoardConfiguration from './components/BoardConfiguration.vue';
+import TokenConfiguration from './components/TokenConfiguration.vue';
+
+Vue.use(Router);
+
+export default new Router({
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: Home,
+      // redirect: { name: 'board'},
+      children: [
```

Comment:
bad router config, this config should contain only the routes for the views, not all the components